book/website/project-design/project-management-methodologies/overview.md

```diff
@@ -0,0 +1,30 @@
+
+Agile frameworks, including Scrum and Kanban, are flexible and can adapt to changing research landscapes and iterative discovery processes that are part and parcel of many research projects.
+
+There are also a number of other methodologies and philosophies that can matter to you: PRINCE2, Lean, Stage/Phase Gate, and many more.
+
+The decision between project management frameworks hinges on the nature of the research, the level of clarity in objectives, and the extent to which the project can accommodate changes in scope or direction.
+Some methodologies are better suited for flexible collaboration between distributed teams.
+Balancing these factors with team dynamics and stakeholder expectations will guide researchers in selecting a framework that aligns with their project's goals, timeline, and complexity, ensuring a smooth and effective research process.
+
+These pages on The Turing Way are designed to assist you in choosing the right project management framework for you.
+
+## Quick comparison between methodologies/frameworks
+
+| **Methodology/Framework** | **Approach** | **Key Features** | **Flexibility** | **Change Management** | **Suitability** | **Origin/typically used in** | **Focus** | **Outcomes** | **Readiness for Use in Research Projects** | **Adaptability for Research Projects** |
+|------------------------|--------------------------------|----------------------------------------------------------|-------------------|--------------------------------|------------------------------------------------------|--------------------------------------------------|--------------------------------------------------|--------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------|
+| **Agile** | Iterative and incremental | Flexibility, adaptability | High | Welcomes changes | Complex projects with variable requirements | Software development, creative projects | Customer satisfaction, team collaboration | Incremental improvements, adaptability to change | Highly suitable due to its iterative nature, allowing for frequent reassessment and adaptation as research findings evolve. | Extremely adaptable, can easily integrate new research directions and findings. |
+| **Scrum** | Iterative, time-boxed sprints | Roles (Scrum Master, Product Owner), Artifacts, "Ceremonies" | Moderate to High | Adapts to changes in sprints | Projects that benefit from regular reassessment | Software development, product development | Team productivity, Iterative development | Deliverables at the end of each sprint | Suitable for research projects that can be broken down into smaller, iterative cycles, with regular reassessment points. | Adaptable within each sprint cycle, but less so between sprints. |
```

Comment:
Noting that this was flagged for inclusive language (scrum master) but I don't think there's any alternative scrum term...

Reply:
Unfortunately that's true, @Arielle-Bennett. The scrum community hasn't taken very well to the idea of changing their language to be more inclusive. I struggle with it myself.

See: https://www.scrum.org/forum/scrum-forum/40445/renaming-master-scrum-master